Makefile

```diff
@@ -26,7 +26,7 @@ local-env: submodule ## Create local environment
 
 submodule: ## Update git submodules
 	git submodule init
-	git submodule update
+	git submodule update --remote --merge
```

Comment:
This might mess things up when there is a zero checkout. The init and update call are fine if there is no module initially, which is the case on CI when this is running. If you want to update, that's when we force a remote and a merge.

Reply:
i had to do this to get the local tests working , i am gonna put it back to what it was before

Reply:
Hmm. That is interesting. Maybe leave it in and we'll see?

Reply:
Oh! I remember why it's that. Because this might cause an involuntary update to running tests locally. Normally, you don't want to force update.

Reply:
i have removed it from the makefile still getting same lint error 😑